### ISSUE TITLE
[Suggestion] Summon from Compendium

### ISSUE THREAD

**Issue by justinrpurdy** [Vauryx/AdvancedSpellEffects#79]
In order to help keep games as performant as possible, the general advice is to avoid having too many extraneous actors in a world. This makes maintaining a folder of all possible summons a concern. Importing the summons from a compendium instead would be great.

**Comment by Vauryx** [comment]
Makes sense. I will give it a shot though I'm heavily reliant on warpgate so I'm not sure how feasible it will be as I'm pretty sure Warpgate only looks for actors in the world. But I'm not certain so I'll give it a try
